src/index.html

```diff
@@ -29,6 +29,61 @@
 		<!-- <link rel="apple-touch-icon" href="favicon-apple.png" />-->
 
 		<title>AntiPlag - Frontend</title>
+
+		<style>
+			#app {
+				width: 100vw;
+				height: 100vh;
+			}
+			svg.spinner {
+				display: block;
+				height: 100%;
+				max-width: 100px;
+				margin: auto;
+				animation: 2s linear infinite spinner-animation;
+			}
+
+			svg.spinner > circle {
+				animation: 1.4s ease-in-out infinite both
+					spinner-circle-animation;
+				display: block;
+				fill: transparent;
+				stroke: #00b5ff;
+				stroke-linecap: round;
+				stroke-dasharray: 283;
+				stroke-dashoffset: 280;
```

Comment:
This is just a spinner, not a progress bar. It animates, therefore the values are animated programatically.

Reply:
No, not at all. Spinner is just an animation, that just spins such as: https://media0.giphy.com/media/3oEjI6SIIHBdRxXI40/200.gif

What you think is a progress bar - and it contains a more complex implementation - it would need to create stream to backend, get the percentage of file and stuff. Not that easy at all.

Spinner is just a fancy animation to tell the user that something is happening and the page did not freeze.

Reply:
Congrats. But I think we did not agreed on not using progressbar when uploading, did we?

Still, tots unrelated to this task. Can we finalize the code review?

Reply:
Yes, thanks good job.